iridium-client-nextjs/src/lib/services/exchange-URL-generator.service.ts

```diff
@@ -0,0 +1,30 @@
+import { CookieService } from './cookie.service';
+import { OauthConstants } from './oauth-constants';
+import { exchange } from './accessToken.service';
+
+export class ExchangeURLGeneratorService {
+    generate() {
+        const queryString = window.location.search;
```

Comment:
`const {search: queryString} = window.location`

Personally I would destructure out search from the object and alias it as queryString if you still want to use that name

Reply:
What value does this add?  Is it just the javascript normal way of doing things?  Or are there other benefits I'm missing?